src/common/utils/index.ts

```diff
@@ -119,3 +119,15 @@ export function isValidEmail(emailAddress: string) {
     emailAddress && emailAddress.length > 1 && EMAIL_REGEX.test(emailAddress)
   );
 }
+
+// Returns unformatted percent in decimal form
+export function getPercentChange(numA: number, numB: number): number | null {
+  const validParams = _.isNumber(numA) && _.isNumber(numB) && numA !== 0;
```

Comment:
you need `_.isFinite`, because `_.isNumber(NaN)` is `true` (also `_.isNumber(Infinity)` and `_.isNumber(-Infinity)`)

Reply:
i kind of feel like numA as the denominator (and therefore representing the initial # ) makes more sense? gonna keep as is for now